drivers/sensor/bme680/bme680.c

```diff
@@ -0,0 +1,444 @@
+/* bme680.c - Driver for Bosch Sensortec's BME680 temperature, pressure,
+ * humidity and gas sensor
+ */
+
+/*
+ * Copyright (c) 2016, 2017 Intel Corporation
+ * Copyright (c) 2017 IpTronix S.r.l.
+ * Copyright (c) 2018 Bosch Sensortec GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <sensor.h>
+#include <init.h>
+#include <gpio.h>
+#include <misc/byteorder.h>
+#include <misc/__assert.h>
```

Comment:
ok

Reply:
Since when did this become a code style rule?

If there is one implicit rule usually, it's the local headers that are included the last. But that's all.